lib/screens/navigate/my_inventory.dart

```diff
@@ -0,0 +1,154 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:universal_io/io.dart';
+import '../navigate/navigate.dart';
+import 'package:flutter_excellence_hr/widgets/navigate/imp_notes_inventory.dart';
+import 'package:flutter_excellence_hr/resources/app_colors.dart';
+import 'package:flutter_excellence_hr/widgets/navigate/comment_section.dart';
+
+class YourInventory extends StatelessWidget {
+  @override
```

Comment:
@Shaktiexcellence change class to proper name

Reply:
changed